### PR TITLE
Updated nginx to pull stable version

### PR DESCRIPTION
**Pull request by pauld-msft**
Updated the nginx installation to use the official nginx stable version, as documented in https://docs.nginx.com/nginx/admin-guide/installing-nginx/installing-nginx-open-source/#installing-prebuilt-debian-packages

Sizes (old on top, new below):
image

### CODE REVIEW

**Review by cormacpayne — DISMISSED** [review]
:shipit:

**Review by daniv-msft — DISMISSED** [review]
Looks good to me! Thanks

File: images/runtime/php-fpm/nginx_conf/nginx.conf [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        access_log /var/log/nginx/access.log;
+        error_log /var/log/nginx/error.log;
+
+        ##
+        # Gzip Settings
+        ##
+
+        gzip on;
+
+        # gzip_vary on;
```
Comment (william-msft) [inline images/runtime/php-fpm/nginx_conf/nginx.conf:48]: nit: we should remove the commented code if we don't plan on using it

**Review by william-msft — APPROVED** [review]

**Review by cormacpayne — APPROVED** [review]

File: images/runtime/php-fpm/template.base.Dockerfile
Comment (waliMSFT) [inline images/runtime/php-fpm/template.base.Dockerfile:1]: Do we need the `nginx` only for the php-fpm (not for php)?

Comment (pauld-msft) [reply images/runtime/php-fpm/template.base.Dockerfile:1]: Correct, PHP uses Apache

**Pull request merged**